tests/vxingest/netcdf_to_cb/test_int_metar_obs_netcdf.py

```diff
@@ -34,19 +35,35 @@ def stub_worker_log_configurer(queue: Queue):
 def setup_connection():
     """test setup"""
     _vx_ingest = VXIngest()
-    _vx_ingest.credentials_file = (os.environ["CREDENTIALS"],)
+    # Ensure credentials_file is a string, not a tuple
+    credentials = os.environ["CREDENTIALS"]
+    if isinstance(credentials, tuple):
+        credentials = credentials[0]
+    _vx_ingest.credentials_file = credentials
```

Comment:
`os.environ['ENV_VAR']` shouldn't be returning tuples - it's supposed to be a dictionary. Do you recall what was going on here that required this?

Reply:
no, I dont recall. I do agree that it shouldn't be a tuple so I will change it.